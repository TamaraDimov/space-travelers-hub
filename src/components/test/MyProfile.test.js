import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import myProfile from '../../page/MyProfile';

test('myProfile Page renders correctly', () => {
  const component = renderer.create(
    <MemoryRouter>
      <myProfile />
    </MemoryRouter>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
