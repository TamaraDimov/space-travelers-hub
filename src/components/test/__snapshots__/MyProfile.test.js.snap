// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`myProfile Page renders correctly 1`] = `<myProfile />`;
